dhall/src/Dhall/Import.hs

```diff
@@ -1119,3 +1121,50 @@ instance Show ImportResolutionDisabled where
 assertNoImports :: MonadIO io => Expr Src Import -> io (Expr Src Void)
 assertNoImports expression =
     Dhall.Core.throws (traverse (\_ -> Left ImportResolutionDisabled) expression)
+
+{-| This function is used by the @--transitive@ flag of the
+    @dhall {freeze,format,lint}@ subcommands to determine which dependencies
+    to descend into
+-}
+dependencyToFile :: Status -> Import -> IO (Maybe FilePath)
```

Comment:
I think a doctest or an example would help make it clearer what this function does.